firebase-firestore/src/androidTest/java/com/google/firebase/firestore/ServerTimestampTest.java

```diff
@@ -181,6 +181,12 @@ public void testServerTimestampsCanReturnPreviousValue() {
     DocumentSnapshot previousSnapshot = accumulator.awaitRemoteEvent();
     verifyTimestampsAreResolved(previousSnapshot);
 
+    // The following update includes an update of tge nested map "deep", which updates it to contain
```

Comment:
tge => the

You might consider configuring your IDE to make typos more prominent. 😛 
![image](https://user-images.githubusercontent.com/206364/60201093-61361d80-97fc-11e9-8e9f-884b900cb268.png)